packages/icons-workflow/README.md

```diff
@@ -0,0 +1,106 @@
+## Description
+
+Spectrum Workflow Icons delivered in a mutable template tag so that they can be leveraged across various frameworks. The default export of this package pre-applies the `html` template tag from `lit-html` for ease of use in the Spectrum Web Components library. Continue reading below for information on using these icons in projects powered by other frameworks.
+
+### Installation
+
+```
+npm install @spectrum-web-components/icons-workflow
+
+# or
+
+yarn add @spectrum-web-components/icons-workflow
+```
+
+### Usage
+
+With the default exports of the packages prepared with the `html` template tag from `lit-html`, the default value of an icon export will be as follows:
+
+```js
+import { LitElement, html } from 'lit-element';
+import '@spectrum-web-components/icon';
+import { Circle } from '@spectrum-web-components/icons-workflow';
```

Comment:
I think it'd be kind of nice if the icon exports had icon in the name. E.g. `SaveIcon` is a lot less ambiguous than `Save`.

Reply:
@andrewhatran having looked at the PR as well, do you have thoughts no which of the naming approaches would feel most easy to leverage for you? I know we've used the pattern as outlined currently no our team, but don't let that color you're response.

Reply:
I sorta like the idea of suffixing our exports with `Icon`. 

Especially since we have icon names like `App`. 

If I saw this for the first time, it wouldn't be clear to me if `App` were an icon or a high-level web component:

```
render(): TemplateResult {
    return html`
         ${App()}
    `
}
```

Reply:
That makes perfect sense. Thanks for sharing a good example! Updating.